### PR TITLE
Removed check for choices if completer is present

### PR DESCRIPTION
**Pull request by Zylvian**
Had a problem supplying my own Completer, as I had to make a dummy list with a single value to get the check to pass. This commit should let people be able to supply **either** a completer **or** choices for the default completer.

### CODE REVIEW

**Comment by kiancross** [comment]
@Zylvian Checks are failing because you need to run `black questionary` over the code, to ensure it is properly formatted.

File: questionary/prompts/autocomplete.py [hunk 1]
```diff
@@ -166,7 +164,12 @@ def get_meta_style(meta: Dict[Text, Any]):
 
     validator = build_validator(validate)
 
+
```
Comment (tmbo) [inline questionary/prompts/autocomplete.py:167]:
```suggestion
```

**Review by tmbo — APPROVED** [review]

File: questionary/prompts/autocomplete.py [hunk 2]
```diff
@@ -149,8 +149,6 @@ def autocomplete(
         Question: Question instance, ready to be prompted (using `.ask()`).
     """
 
-    if not choices:
-        raise ValueError("No choices is given, you should use Text question.")
 
     merged_style = merge_styles([DEFAULT_STYLE, style])
```
Comment (kiancross) [inline questionary/prompts/autocomplete.py:154]:
```suggestion
```

File: questionary/prompts/autocomplete.py [hunk 3]
```diff
@@ -167,6 +165,10 @@ def get_meta_style(meta: Dict[Text, Any]):
     validator = build_validator(validate)
 
     if completer is None:
+
```
Comment (kiancross) [inline questionary/prompts/autocomplete.py:168]:
```suggestion

```

File: questionary/prompts/autocomplete.py [hunk 4]
```diff
@@ -167,6 +165,10 @@ def get_meta_style(meta: Dict[Text, Any]):
     validator = build_validator(validate)
 
     if completer is None:
+        
+        if not choices:
+            raise ValueError("No choices is given, you should use Text question.")
+
```
Comment (kiancross) [inline questionary/prompts/autocomplete.py:171]:
```suggestion

```

File: questionary/prompts/autocomplete.py [hunk 5]
```diff
@@ -149,8 +149,6 @@ def autocomplete(
         Question: Question instance, ready to be prompted (using `.ask()`).
     """
 
-    if not choices:
-        raise ValueError("No choices is given, you should use Text question.")
```
Comment (kiancross) [inline questionary/prompts/autocomplete.py:152]:
```suggestion
```

**Review by kiancross — CHANGES_REQUESTED** [review]

**Review by kiancross — APPROVED** [review]

**Pull request merged**